src/utils.rs

```diff
@@ -119,3 +119,10 @@ pub fn parse_args(
             )
     }
 }
+
+/// Whether or not Wasmer should print with color
+pub fn wasmer_should_print_color() -> bool {
+    std::env::var("WASMER_DISABLE_COLOR")
+        .map(|_| false)
+        .unwrap_or(true)
```

Comment:
`.unwrap_or(stdout_isatty())` from the `isatty` crate?

Which also implies that it shouldn't be `WASMER_DISABLE_COLOR` because we might also want to forcibly enable color when isatty is false. Perhaps `WASMER_COLORS` which may be true or false.